### PR TITLE
Add account properties

### PR DESCRIPTION
**Pull request by 0nkery**

### CODE REVIEW

**Review by manifest — APPROVED** [review]

File: src/app/api/v1/class/mod.rs [hunk 1]
```diff
@@ -53,6 +58,32 @@ impl ClassResponseBody {
         self.position = Some(position_secs);
     }
 
+    pub fn filter_class_properties(&mut self, keys: &Vec<String>) {
```
Comment (mgrachev) [inline src/app/api/v1/class/mod.rs:61]:
```suggestion
    pub fn filter_class_properties(&mut self, keys: &[String]) {
```

File: src/app/api/v1/class/mod.rs [hunk 2]
```diff
@@ -53,6 +58,32 @@ impl ClassResponseBody {
...
+            }
+        }
+
+        self.properties = props;
+    }
+
+    pub fn set_account_properties(
+        &mut self,
+        mut account_properties: KeyValueProperties,
+        keys: &Vec<String>,
```
Comment (mgrachev) [inline src/app/api/v1/class/mod.rs:76]:
```suggestion
        keys: &[String],
```

File: src/app/api/v1/account.rs [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+    Extension(ctx): Extension<Arc<dyn AppContext>>,
+    Path(property_id): Path<String>,
+    AccountIdExtractor(account_id): AccountIdExtractor,
+) -> AppResult {
+    ReadProperty {
+        ctx: ctx.as_ref(),
+        account_id: &account_id,
+        property_id,
+    }
+    .run()
```
[the remark this answers is not in the document]
Comment (0nkery) [reply src/app/api/v1/account.rs:28]: It's like 'read_property_do_read_property`.

File: src/app/api/v1/account.rs [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+    id: &AccountId,
+) -> anyhow::Result<crate::db::account::Object> {
+    let account = {
+        let mut conn = state.get_conn().await?;
+        crate::db::account::ReadQuery::by_id(id)
+            .execute(&mut conn)
+            .await?
+            .ok_or_else(|| anyhow!("Failed to find account"))?
+    };
+    Ok(account)
```
[the remark this answers is not in the document]
Comment (0nkery) [reply src/app/api/v1/account.rs:110]: No benefit, will remove. :)

File: src/app/api/v1/account.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (0nkery) [reply src/app/api/v1/account.rs:28]: I can't name this method 'do.'

**Review by manifest — APPROVED** [review]

**Pull request merged**